hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/has/isauthorizedraw/IsAuthorizedRawCall.java

```diff
@@ -83,6 +83,7 @@ public IsAuthorizedRawCall(
         this.messageHash = requireNonNull(messageHash);
         this.signature = requireNonNull(signature);
         this.customGasCalculator = requireNonNull(customGasCalculator);
+        requireNonNull(attempt.signatureVerifier());
```

Comment:
Why is this necessary for isAuthorizedRaw?

Reply:
To be fixed #16448